Apps/Src/ReadCarCAN.c

```diff
@@ -68,27 +68,24 @@ static void updateSaturation(int8_t chargeMessage){
     chargeMsgSaturation = newSaturation;
 }
 
+// exception struct callback for charging disable, kills contactors and turns of display
+static void callback_disableContactors(void){
+    // Kill contactors 
+    Contactors_Set(ARRAY_CONTACTOR, OFF, true);
+    Contactors_Set(ARRAY_PRECHARGE, OFF, true);
+
+    // Turn off the array contactor display light
+    UpdateDisplay_SetArray(false);
```

Comment:
Does this matter if we are on the EVAC screen? Also, just put the `Display_Evac(SOC, SBPV); // Display evacuation message` code line in this callback

Reply:
Good point- I don't know too much about how the display works, but it probably doesn't matter. I believe this is changed in the ReadCarCAN PR (sorry, I should have warned you that most of the fault handling update for ReadCarCAN happens elsewhere), so it should be fine here.